Automattic/jetpack-start
# Add a step to ask if the user has an existing store

Adds a new step after returning from setting up WooCommerce. The positioning of this step is a little awkward - you'll see it if you "Install WooCommerce" then opt out of the WC wizard, or if we detect you already have WC installed.

![screen shot 2016-11-10 at 3 24 47 pm](https://cloud.githubusercontent.com/assets/541093/20194624/174213c0-a761-11e6-873f-0ae9d9d3c742.png)

To test:

- Check out this branch
- Go through the business flow
- Install WooCommerce, then opt-out of the WC wizard
- OR go through the WC wizard, and click "back to dashboard" at the end (small grey text under the box)
- You should return to this step, where you can select a store - selecting one automatically sends the store to the completed step event tracking, and takes you to the review.
- Clicking No goes directly to the review.
- Clicking Next also just takes you to the review, without sending an event, since you haven't selected anything.

@jessefriedman @johnHackworth @roccotripaldi @tyxla
